pkg/virt-controller/watch/vm.go

```diff
@@ -1824,7 +1824,7 @@ func setupStableFirmwareUUID(vm *virtv1.VirtualMachine, vmi *virtv1.VirtualMachi
 		return
 	}
 
-	vmi.Spec.Domain.Firmware.UUID = types.UID(uuid.NewSHA1(firmwareUUIDns, []byte(vmi.ObjectMeta.Name)).String())
+	vmi.Spec.Domain.Firmware.UUID = types.UID(uuid.NewSHA1(firmwareUUIDns, []byte(vmi.Namespace+vmi.Name)).String())
```

Comment:
:+1: 

Would it make sence to add a unit test? Perhaps extending this one:

https://github.com/kubevirt/kubevirt/blob/f2fcb8c57ee71b0440746f2bda4f528d279fcf83/pkg/virt-controller/watch/vm_test.go#L2053

Reply:
Done

Reply:
Thanks!